sdks/java/fn-execution/src/main/java/org/apache/beam/sdk/fn/channel/ManagedChannelFactory.java

```diff
@@ -41,7 +42,24 @@ public static ManagedChannelFactory createEpoll() {
     return new Epoll();
   }
 
-  public abstract ManagedChannel forDescriptor(ApiServiceDescriptor apiServiceDescriptor);
+  public final ManagedChannel forDescriptor(ApiServiceDescriptor apiServiceDescriptor) {
+    return forDescriptorOnly(apiServiceDescriptor);
```

Comment:
Shall we get away with this indirection?
Implementer can always overwrite this public method (if we drop the final).